content/components/responses.mdx

```diff
@@ -10,24 +10,20 @@ The `<ResponseField>` component is designed to define the return values of an AP
   A response field example
 </ResponseField>
 
-<RequestExample>
-
-```jsx ResponseField Example
+```jsx
 <ResponseField name="response" type="string" required>
   A response field example
 </ResponseField>
 ```
 
-</RequestExample>
-
 ## Props
 
 <ResponseField name="name" type="string" required>
   The name of the response value.
 </ResponseField>
 
 <ResponseField name="type" type="string" required>
-  Expected type of the response value
+  Expected type of the response value, you can also pass a custom type string
```

Comment:
what is this addition for? are you trying to say...

```suggestion
  Expected type of the response value - this can be any arbitrary string.
```

Reply:
yah much better